src/main/kotlin/com/github/devcordde/devcordbot/core/autohelp/ContentFetcher.kt

```diff
@@ -92,13 +92,17 @@ class ContentFetcher(
 
     private suspend fun fetchAttachment(attachment: Message.Attachment): String {
         val stream = attachment.retrieveInputStream().await()
```

Comment:
Instead of doing this and closing it manually at the end, the `use` extension function should be used.